tests/unit/test_simulation.py

```diff
@@ -482,27 +481,25 @@ def test_save_load(self):
                 sim.save(test_name)
 
     def test_load_param(self):
-        # Test load_sim for parameters imports
-        filename = f"{uuid.uuid4()}.p"
-        model = pybamm.lithium_ion.SPM()
-        params = pybamm.ParameterValues("Chen2020")
-        sim = pybamm.Simulation(model, parameter_values=params)
-        sim.solve([0, 3600])
-        sim.save(filename)
-
-        try:
-            pkl_obj = pybamm.load_sim(os.path.join(filename))
-        except Exception as excep:
-            os.remove(filename)
-            raise excep
-
-        assert (
-            "graphite_LGM50_electrolyte_exchange_current_density_Chen2020"
-            == pkl_obj.parameter_values[
-                "Negative electrode exchange-current density [A.m-2]"
-            ].__name__
-        )
-        os.remove(filename)
+        with tempfile.NamedTemporaryFile(delete=False) as tmp:
```

Comment:
Could you please use [pytest's `tmp_path` fixture](https://docs.pytest.org/en/stable/how-to/tmp_path.html) here instead?